src/utils/EventUtils.ts

```diff
@@ -73,9 +74,15 @@ export function canEditOwnEvent(mxEvent: MatrixEvent): boolean {
 }
 
 const MAX_JUMP_DISTANCE = 100;
-export function findEditableEvent(room: Room, isForward: boolean, fromEventId: string = undefined): MatrixEvent {
-    const liveTimeline = room.getLiveTimeline();
```

Comment:
That line confuses me a lot.
@bwindels could you fill in the gap for me please. How come we only read events from the live timeline. What happens if you load a very old set event by clicking a matrix.to link. That will put all of those events in another timeline, and presumably you won't be able to select an event to edit using arrow keys.

Would you say that's accurate, or am I missing the obvious here?

Reply:
tbh I don't remember really how permalinks are handled in the timeline code, and at this point you're probably more aware than I am :) But I don't remember specifically testing this with permalinks, so it's likely I didn't consider this use case.

Reply:
@gsouquet I guess we need to make a ticket to check this out and handle this case separately.

Reply:
I'll do a bit more testing and try to up my understanding
No need to create a ticket, this code has been running there for sufficiently long that it should be ok